strictdoc/backend/sdoc/models/document_view.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Optional
+
+from strictdoc.backend.sdoc.models.type_system import (
+    ViewElementTags,
+    ViewElementHiddenTag,
+)
+
+from strictdoc.helpers.auto_described import auto_described
+
+
+@auto_described()
+class ViewElement:
+    def __init__(
+        self,
+        id: str,
+        tags: List[ViewElementTags],
+        hidden_tags: Optional[List[ViewElementHiddenTag]] = None,
+        name: Optional[str] = None,
+    ):
+        self.id = id
```

Comment:
It is better not to use `id` because it is a Python built-in and some quality tools complain that you overlap with Python's id:

```
strictdoc-project/strictdoc (html) $ python
Python 3.7.15 (default, Dec 29 2022, 19:39:20)
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> id
<built-in function id>
```

In this case, `view_id` would do just fine.